### ISSUE TITLE
Bug- While using Twemoji SVG.

### ISSUE THREAD

**Issue by naveen521kk** [appfigures/react-easy-emoji#9]
I have used `emoji("👋" ,{baseUrl:"https://twemoji.maxcdn.com/2/svg/",ext:".svg",size:""}) ` function for loading emoji but I have an empty image near the emoji while the site loads. Like the below picture.
![error](https://user-images.githubusercontent.com/49693820/78984533-13e52b80-7b44-11ea-9468-2e9f7001acd8.PNG) .
On inspecting the element I found the below one.
```html
<img alt="👋" draggable="false" src="http:https://twemoji.maxcdn.com/2/svg//1f44b.svg" style="height: 1em; width: 1em; margin: 0px 0.05em 0px 0.1em; vertical-align: -0.1em;">
```

I found that `http` has been added in front of `https://twemoji.maxcdn.com/2/svg//1f44b.svg` which should not happen. Please fix this.

**Comment by oztune** [comment]
Thanks for the bug report @naveen521kk! We are working on a fix now.

**Comment by oztune** [comment]
@naveen521kk Just a quick update: The library expects you to pass the protocol as a separate field. So you can change your code to:
```
emoji("👋" ,{baseUrl:"//twemoji.maxcdn.com/2/svg/", protocol: "https:", ext:".svg",size:""})
```

Also, if your page is already served via https you can just drop the protocol all-together as it will used automatically.

**Comment by naveen521kk** [comment]
Then, please update the ReadMe file in the SVG.

**Comment by oztune** [comment]
Even better, I changed the default so that your example above will just work! It's now published to v1.4.0.

Stay safe, wash your ✋
